docs/usage.md

```diff
@@ -341,15 +611,15 @@ properties:
 The raw schema definition can be difficult to understand without experience
 working with JSON Schema specifications. For this reason, a feature was included
 which can generate user-friendly HTML documentation for the input file schema
-which can be browsed with the user's preferred web browser. 
+which can be browsed with the user's preferred web browser.
```

Comment:
I would reword lines 612 through 614 as the following:
```suggestion
working with JSON Schema specifications. For this reason, a feature is included
that generates user-friendly HTML documentation for the input file schema
and can be browsed with your preferred web browser.
```

Reply:
the re-wording says "a feature ... can be browsed with your preferred web browser" which is confusing IMO. Perhaps splitting into two sentences would address your concern as well. E.g:

```
The raw schema definition can be difficult to understand without experience
working with JSON Schema specifications. For this reason, a feature is included
that generates user-friendly HTML documentation for the input file schema. This
HTML documentation can be browsed with your preferred web browser.
```